web/jsonapi/data.go

```diff
@@ -0,0 +1,85 @@
+package jsonapi
+
+import (
+	"encoding/json"
+
+	"github.com/cozy/cozy-stack/couchdb"
+)
+
+// Object is an interface to serialize something to a JSON-API Object
+type Object interface {
+	couchdb.Doc
+	SelfLink() string
+	Relationships() RelationshipMap
+	Included() []Object
+}
```

Comment:
Nope. I think it makes sense to put it in `jsonapi.Object` to make it clear that we are talking about the serialisation for the public API side (and not couch for instance)...